config/sync/migrate_plus.migration.users.yml

```diff
@@ -0,0 +1,54 @@
+uuid: 5a897396-9210-4c79-b877-2e3107ded0c4
+langcode: en
+status: true
+dependencies: {  }
+id: users
```

Comment:
Couldn't recycle the d7_users.yml core migration, as it griped about an incompatible core version (??). This works well and means the migrations are also neatly tagged/grouped and all the d6/d7 upgrade cruft can be ignored/filtered out.